docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -11,6 +11,20 @@ endif::[]
 [id="{p}-ga-upgrade"]
 == Upgrade to ECK {eck_version}
 
+[IMPORTANT]
+================================
+This is the upgrade guide for ECK. For Elastic Stack upgrade, please refer to this <<{p}-upgrading-stack,guidance>> for more details.
+================================
+
+[CAUTION]
+================================
+Upgrading the ECK operator results in a one-time update to existing managed resources in the cluster. This triggers a rolling restart of pods, on which Elastic Stack components are running, by Kubernetes to apply those changes. Therefore, if you have a large Elasticsearch cluster or multiple Elastic Stack deployments, this rolling restart will cause performance degradation, which might be disruptive or inconvenient.
+
+For production use, when you plan to upgrade ECK, it's important to include this pods rolling restart into time schedule consideration.
```

Comment:
```suggestion
For production use, when you plan to upgrade ECK, it's important to include this rolling restart into time schedule consideration.
```
I am a bit confused by this statement. Why are time schedule consideration a concern here? Do we mean that the overall time the upgrade takes is the time to upgrade the ECK operator plus the time to roll all managed workloads/Elasticserach clusters? If so, we should maybe be explicit.

Reply:
> Do we mean that the overall time the upgrade takes is the time to upgrade the ECK operator plus the time to roll all managed workloads/Elasticserach clusters

Yes, your reading is correct.
I will make it explicit.